ksy_style_guide.adoc

```diff
@@ -106,7 +106,7 @@ different purposes.
 
 For top-level meta section, use the following order of keys:
 
-. `id` — main identifier
+. `id` — main identifier, should match the name of the `.ksy` file
```

Comment:
Totally makes sense, but please use `MUST` instead of `should`, as this is actually a pretty hard requirement.

Reply:
(and it is `MUST`, not `must`, as we're reffering to RFC 2119 terms).

Reply:
Certainly! This change has been added to the pull request.